src/client/Microsoft.Identity.Client/Core/Constants.cs

```diff
@@ -14,6 +14,7 @@ internal static class Constants
 
         public const string UapWEBRedirectUri = "https://sso"; // only ADAL supports WEB
         public const string DefaultRedirectUri = "urn:ietf:wg:oauth:2.0:oob";
+        public const string MobileDefaultRedirectUri = "msal4a1aa1d5-c567-49d0-ad0b-cd957a47f842://auth"; // in msidentity-samples-testing tenant -> PublicClientSample
```

Comment:
Why are we storing this constant in the MSAL library ? This seems to be used exclusively in a dev app.

Reply:
ugh...yes. my oversight. i'll move it to the test constants.